tornettools/generate_tgen.py

```diff
@@ -393,9 +393,14 @@ def get_clients(args):
 def __get_perf_clients(args, n_exit_users, n_hs_users):
     perf_clients = []
 
-    # split the perf clients between onion-service clients and non-onion-service (exit) clients
-    n_hs_perf = __round_or_ceil(n_hs_users * args.torperf_scale)
-    n_exit_perf = __round_or_ceil(n_exit_users * args.torperf_scale)
+    if args.torperf_scale:
+        # Legacy way
+        n_hs_perf = __round_or_ceil(n_hs_users * args.torperf_scale)
+        n_exit_perf = __round_or_ceil(n_exit_users * args.torperf_scale)
+    else:
+        n_hs_perf = args.torperf_n
+        n_exit_perf = args.torperf_n
```

Comment:
This will result in 200 perf clients total (100 hs + 100 exit), which is unintuitive when setting `torperf_n=100`. Do we want separate knobs to tune the hs and exit clients, to enable more precise network generation? For example, `torperf_num_exit` and `torperf_num_onionservice`?

Reply:
My thinking is that generally we'll want the same amount of data/confidence in both types, and that accidentally ending up with more total perf clients than you thought you were getting is a less-bad footgun than ending up with different numbers of perf clients for each type.

OTOH I agree it'd be useful to have some way of controlling them separately, particularly to set either to zero.

Maybe have it as separate options, but log a warning if they are different and both non-zero?